pyproject.toml

```diff
@@ -7,14 +7,14 @@ dependencies = [
     "attrs",
     "catkin-pkg==1.0.0 ; sys_platform == 'linux'",
     "click==8.1.8",
-    "craft-application[remote]~=4.10",
+    "craft-application[remote] @ git+https://github.com/canonical/craft-application@main",
```

Comment:
is this not pointing to `5.0.0` on purpose?

Reply:
updated